### DISCUSSION TITLE
Many to many with intermediate table

### DISCUSSION THREAD

**Discussion by Xazgl** [prisma/prisma#18225] [Q&A]
How to add a car to favorites via connectOrCreate (similar to the basket). I have a connection between Car model and SessionClient model many to many. The machines already exist in the database, customers receive a token when they log in on the site. Clients model and 
FavoriteCar model intermediate table. I need add Car in favorite 
![image](https://user-images.githubusercontent.com/86427773/223034015-c1080c2a-0c3e-4265-8834-8ba39ad3be81.png)
Car model 
![image](https://user-images.githubusercontent.com/86427773/223034342-2364d263-6adc-403e-82b2-36ba4a50152f.png)

**Comment by nurul3101** [comment]
Hello @Xazgl 👋 

Do you mind sharing the models in text format? It would become much easier to copy your models locally and write a query for it. You can actually directly create a record on the connecting table (FavoriteCarsToCar), and that would connect the two entities.

**Reply by nurul3101** [reply]
@Xazgl Do let us know in case you still need any assistance.
